packages/code.rb

```diff
@@ -17,6 +17,7 @@ class Code < Package
   end
 
   binary_url ({
+    armv7l:'https://github.com/stevedesmond-ca/vscode-arm/releases/download/1.28.0/vscode-1.28.0.deb'
```

Comment:
I don't think you will want to this with a .deb package.  This is reserved for Chromebrew packages only.